Apollo/Binary/Encoder.cs

```diff
@@ -100,12 +104,40 @@ public static class Encoder {
             
             writer.Write(Preferences.Time);
         });
+        
+        public static void EncodeAnything<T>(BinaryWriter writer, dynamic o) {      
+            Type oType = typeof(T);          
+            
+            if(Nullable.GetUnderlyingType(typeof(T)) != null) {
+                writer.Write(o != null);
+                if(o != null) writer.Write(o);
```

Comment:
This will fail for nullables of reference types, but VS warns that you shouldn't use them anyways

Reply:
oh well